package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmrc-frontend",
-  "version": "6.28.0",
+  "version": "6.29.0",
```

Comment:
if you set this to 6.28.0 and then run
```
npm version minor --git-tag-version=false
```
it will update this again and update the version number in package-lock.json which also needs to be updated - (suggest doing like this as I can't remember all the places in package-lock.json it is, think there migth be more than one so manually updating it might leave it partly incorrect)

Reply:
we've set git-tag-version=false in .npmrc but just in case it doesn't apply I've included it here

Reply:
Done 👍